Source/common/SNTBlockMessage.m

```diff
@@ -177,6 +181,10 @@ + (NSURL *)eventDetailURLForEvent:(SNTStoredEvent *)event customURL:(NSString *)
       ^{ return event.fileSHA256 ? event.fileBundleHash ?: event.fileSHA256 : nil; },
                                                  @"%bundle_or_file_identifier%",
       ^{ return event.executingUser; },          @"%username%",
+      ^{ return event.fileBundleID; },           @"%file_bundle_id%",
```

Comment:
I think it would be best to call this one `%bundle_identifier%` to match the pattern of `file_identifier` and `bundle_or_file_identifier`

Reply:
Doesn't sound like there's a great option, we've overloaded identifier. The existing `_identifier` values should really be `_hash` but renaming them now is a pain.